### ISSUE TITLE
Mob Leashing Timer Fails To Refresh

### ISSUE THREAD

**Issue by YourNewfriend** [ccshiro/cc-buglist#1150]
Currently, a mob's leash timer isn't refreshed by missed attacks.  I haven't tested other methods (parry/dodge/immune/resist), but I don't believe this should be the case, because like this, if a player gets unlucky enough with his attacks, a mob could end up fleeing in the middle of combat.

In this video, kiting with a spell that's treated as immune is done: https://youtu.be/SgLNayOzWoc?t=2m54s

**Comment by ccshiro** [comment]
I don't think it can be assume to work the same for missed just because immune works like that.

Basically:
a) It's broken if immuned kiting doesn't work currently (needs testing)
b) Or it needs evidence about missed kiting.
